lntest/itest/lnd_test.go

```diff
@@ -4718,18 +4688,13 @@ func testListChannels(net *lntest.NetworkHarness, t *harnessTest) {
 	const bobRemoteMaxHtlcs = 100
 
 	// Create two fresh nodes and open a channel between them.
-	alice, err := net.NewNode("Alice", nil)
-	if err != nil {
-		t.Fatalf("unable to create new node: %v", err)
-	}
+	alice := net.NewNode(t.t, "Alice", nil)
 	defer shutdownAndAssert(net, t, alice)
 
-	bob, err := net.NewNode("Bob", []string{
-		fmt.Sprintf("--default-remote-max-htlcs=%v", bobRemoteMaxHtlcs),
-	})
-	if err != nil {
-		t.Fatalf("unable to create new node: %v", err)
-	}
+	bob := net.NewNode(
+		t.t, "Bob", []string{
+			fmt.Sprintf("--default-remote-max-htlcs=%v", bobRemoteMaxHtlcs),
+		})
```

Comment:
micro nit: put closing parenthesis on new line, here and below.

Reply:
done!